db/re/item_db.txt

```diff
@@ -10791,6 +10793,7 @@
 20825,Para_Team_Manteau160,Awakened Eden Group Manteau III,4,0,,0,,30,,0,0xFFFFFFFF,63,2,4,,160,1,0,{ bonus2 bSubEle,Ele_All,10; bonus bFlee,12; bonus bMaxHP,1000; bonus bFlee2,2; .@r = getrefine(); if(.@r >= 7){ bonus2 bSubEle,Ele_Neutral,5; if(.@r >= 9){ skill "BS_GREED",1; } } },{},{}
 20826,Angel_Feather,Angel Feather,4,0,,200,,18,,1,0xFFFFFFFF,63,2,4,,1,1,18,{},{},{}
 20830,Wings_of_Seraph,Wings of Seraph,4,0,,200,,18,,1,0xFFFFFFFF,63,2,4,,1,1,8,{},{},{}
+22984,Kahluna_Milk,Kahluna Milk,3,6,,10,,,,,,,,,,,,,{},{},{}
```

Comment:
I don't know if we have something like this already, but it should have this effect:
"Restores 10 HP every 10 seconds for 3 minutes."
Basically I would say this sounds like a SC(like a stat food).